package.json

```diff
@@ -64,9 +63,11 @@
     "vue-mq": "^1.0.1",
     "vue-property-decorator": "^7.0.0",
     "vue-router": "^3.0.1",
-    "vue-select": "^2.5.1",
     "vue-slider-component": "^2.8",
     "vuejs-paginate": "^2.1.0"
   },
-  "types": "./index.d.ts"
+  "types": "./index.d.ts",
+  "dependencies": {
+    "vue-multiselect": "^2.1.6"
```

Comment:
dependencies should be added in devDependencies and peerDependencies objects - not regular dependencies